content/en/registry/instrumentation-js-contrib-fastify.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Fastify OpenTelemetry
+registryType: instrumentation
+isThirdParty: true
+language: js
+tags:
+  - Node.js
+  - fastify
+	- fastify-plugin
```

Comment:
Could you replace the tab at the start of line 9 with two spaces?

> <img width="365" alt="image" src="https://user-images.githubusercontent.com/4140793/156181926-8d4650a9-43ce-497f-ae32-3a483c6868a8.png">

Reply:
should be fixed